Cargo.toml

```diff
@@ -47,11 +47,13 @@ futures-util = { version = "0.3.14", features = ["io"] }
 futures-executor = "0.3.14"
 hex = "0.4.0"
 hmac = "0.10.1"
+hyper = { version = "0.14", default-features = false, features = ["client", "http1", "tcp"], optional = true }
```

Comment:
Would you mind updating this to be a block (i.e. `[dependencies.hyper]` rather than the single line here) for readability?